### PR TITLE
Step3 json

### PR DESCRIPTION
**Pull request by jdstriker731**
https://dev-dot-johndallard-step-2020.uc.r.appspot.com
^ The link to the version of my site for Step 3 - JSON

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -63,3 +64,21 @@ const addContentToDOM = content => {
 
   quoteContainer.innerText = content;
 };
+
+/** 
+ * Prints a random message to the DOM using JSON sent from DataServlet.java.
+ */
+const fetchMessageUsingJSON = () => {
+  fetch('/data').then(response => response.json()).then( messagesObj => {
```
Comment (zkoogle) [inline portfolio/src/main/webapp/script.js:72]: nit: space.
+askewc@: is there any auto-lint or auto-formatter?

Comment (jdstriker731) [reply portfolio/src/main/webapp/script.js:72]: I'm confused. Where do you want to see the space?

Comment (zkoogle) [reply portfolio/src/main/webapp/script.js:72]: After the second "then("

Comment (jdstriker731) [reply portfolio/src/main/webapp/script.js:72]: In the examples they gave us it was in all one line, but I'll change it

Comment (jdstriker731) [reply portfolio/src/main/webapp/script.js:72]: const fetchMessageUsingJSON = () => {
  fetch('/data').then(response => response.json()).then( 
    messagesObj => {
    // messagesObj is an object, not a string, so we have to
    // reference its fields to create HTML content
    
    // Get random message from the the "messages" field
    // of messagesObj
    const messagesSize = messagesObj.messages.length;
    const message = messagesObj.messages[Math.floor(Math.random() * messagesSize)];

    // Add message to the page
    factContainer.innerText = message;
  });
};

Comment (jdstriker731) [reply portfolio/src/main/webapp/script.js:72]: `const fetchMessageUsingJSON = () => {
  fetch('/data').then(response => response.json()).then( 
    messagesObj => {
    // messagesObj is an object, not a string, so we have to
    // reference its fields to create HTML content
    
    // Get random message from the the "messages" field
    // of messagesObj
    const messagesSize = messagesObj.messages.length;
    const message = messagesObj.messages[Math.floor(Math.random() * messagesSize)];

    // Add message to the page
    factContainer.innerText = message;
  });
};
`

Is this what you want it to look like?

Comment (jdstriker731) [reply portfolio/src/main/webapp/script.js:72]: `const fetchMessageUsingJSON = () => {
  fetch('/data').then(response => response.json()).then( 
    messagesObj => {
    // messagesObj is an object, not a string, so we have to
    // reference its fields to create HTML content
    
    // Get random message from the the "messages" field
    // of messagesObj
    const messagesSize = messagesObj.messages.length;
    const message = messagesObj.messages[Math.floor(Math.random() * messagesSize)];

    // Add message to the page
    factContainer.innerText = message;
  });
};
`

Comment (jdstriker731) [reply portfolio/src/main/webapp/script.js:72]: I tried posting it here to see how you want it but I keep getting this weird formatting

**Comment by jdstriker731** [comment]
I added the spacing in that function. Please let me know if that's how you want it to be presented.

**Comment by jdstriker731** [comment]
After talking to Cam, I think I now understand what you meant @zkoogle

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (askewc) [reply portfolio/src/main/webapp/script.js]: use three backticks for blocks of code, I bet

**Pull request merged**